db/migrate/20220103143414_add_videoids_to_artists.rb

```diff
@@ -0,0 +1,7 @@
+class AddVideoidsToArtists < ActiveRecord::Migration[6.1]
+    def change
+      add_column :artists, :first_video_id, :string
+      add_column :artists, :second_video_id, :string
+      add_column :artists, :third_video_id, :string
```

Comment:
We should probably aim to use an array here rather than separate video columns

Reply:
working on this!